internal/pkg/cli/svc_init.go

```diff
@@ -321,11 +337,24 @@ func (o *initSvcOpts) askSvcName() error {
 	return nil
 }
 
-// askDockerfile prompts for the Dockerfile by looking at sub-directories with a Dockerfile.
-func (o *initSvcOpts) askDockerfile() error {
-	if o.dockerfilePath != "" {
+func (o *initSvcOpts) askImage() error {
+	if o.image != "" {
 		return nil
 	}
+	image, err := o.prompt.Get(wkldInitImagePrompt, wkldInitImagePromptHelp, nil,
+		prompt.WithFinalMessage("Image:"))
+	if err != nil {
+		return fmt.Errorf("get image location: %w", err)
+	}
+	o.image = image
+	return nil
+}
+
+// askDockerfile prompts for the Dockerfile by looking at sub-directories with a Dockerfile.
+func (o *initSvcOpts) askDockerfile() (useImage bool, err error) {
```

Comment:
it's a bit odd that the method named `askDockerfile` returns a boolean that's true or false for the next step instead of it being about whether the Dockerfile was selected or not.

Instead, what do you think about returning `true, nil` if a Dockerfile was selected. `false, nil` if a custom image was selected, `false, err` on some weird selection error.
```suggestion
// The comment above to explain the bool
func (o *initSvcOpts) askDockerfile() (isDfSelected bool, err error) {
```